README.md

```diff
@@ -1,2 +1,18 @@
 # hw01
 Repository for homework 1
+# Hoi Ying's biography
+###### I am a MAPSS student at Uchicago. I like cooking and ballet dancing. My favorite holiday is **April Fools**.
+[link](https://en.wikipedia.org/wiki/April_Fools%27_Day)
+* The cities I like the best
+   + Chicago
+      - when it doesn't snow
+   + Tokyo
+1. My favorite GOT characters
```

Comment:
Minor thing, but since this part only has 1, it doesn't need to be a numbered list